rest/cluster-pending-folders-delete.rst

```diff
@@ -0,0 +1,19 @@
+DELETE /rest/cluster/pending/folders
+====================================
+
+.. versionadded:: 1.18.0
+
+Remove records about a pending folder, without ignoring it in the
+configuration.  Valid values for the ``folder`` and ``device``
+parameters are those from the corresponding
+:doc:`/rest/cluster-pending-folders-get` endpoint.  The ``device``
+parameter is optional and affects announcements of this folder from
+*any* device if omitted.
```

Comment:
Second part feels backwards: Either "and affects ... from the given device or *any* device if omitted." or just "and omitting it removes this folder from all devices."

Reply:
Committed the first suggestion.